crates/gitbutler-tauri/tauri.conf.release.json

```
@@ -1,55 +1,55 @@
 {
+	"productName": "GitButler",
+	"identifier": "com.gitbutler.app",
 	"build": {
-		"beforeBuildCommand": "[ $CI = true ] || pnpm build:desktop -- --mode production && cargo build --release -p gitbutler-git && bash ./gitbutler-tauri/inject-git-binaries.sh"
+		"beforeBuildCommand": "[ $CI = true ] || pnpm build:desktop -- --mode production && cargo build --release -p gitbutler-git && bash ./gitbutler-tauri/inject-git-binaries.sh && bash ./crates/gitbutler-tauri/inject-relauncher.sh GitButler"
```

```suggestion
		"beforeBuildCommand": "[ $CI = true ] || pnpm build:desktop -- --mode production && cargo build --release -p gitbutler-git && bash ./creates/gitbutler-tauri/inject-git-binaries.sh && bash ./crates/gitbutler-tauri/inject-relauncher.sh GitButler"
```

I ran into this by accident. The `nightly` variant does have this change, but the `release` doesn't. In result it failed to build locally for me. Your Miles May Vary.

---

Hey, thanks for spotting that. I actually noticed it, and there's a couple more changes I need to get in now that Tauri v2 has been released. We should be merging in this PR in the coming week!